src/nft.ts

```diff
@@ -298,7 +268,7 @@ export default class Nft extends FactoryBase {
    * @returns
    * @param {NftSearchParams & SearchOption} searchParams
    */
-  searchCollections(searchParams: NftSearchParams & SearchOption): Promise<NftToken[]> {
+  searchCollections(searchParams: NftSearchParams & SearchOption) {
```

Comment:
is this correct? it seems this function is actually returning a value.

Reply:
A new return type has been defined. There was something that didn't fit with NftToken.